en/docs/get-started/get-started.md

```diff
@@ -0,0 +1,462 @@
+
+5. In your user profile, toggle the **Lock User** switch to unlock your profile.
+
+6. In the email you received from Asgardeo (with the subject **Here is your new account in the organization <ORGANIZATION_ID>**), click **Set Password**.
+
+7. Enter a password that matches the given criteria in the **Enter new password** and **Confirm password** fields, and click **Proceed**.
+
+!!! tip 
+    You can create more users to test your front-end application with multiple users.
+
+#### Step 2.4.4: Consume the REST API via the front-end application
+
+To start the front-end application and send requests to the `readingList` REST API via it, follow the steps given below:
+
+1. In the terminal, navigate to the `choreo-reading-list-application-example/reading-list-front-end` directory in the clone of your GitHub repository.
+
+2. Issue the following commands in the given order:
+
+    `npm i`
+
+    `npm run dev`
+
+    The front-end application will start running at http://localhost:5173/.
+
+3. Access the front-end application via http://localhost:5173/.
+
+4. Click Login, and sign in with the credentials of a user that you have created in Asgardeo.
+
+5. To allow your application to access your Choreo account, select the **User Account** checkbox and click **Allow**.
+
+    The application opens as follows.
+
+    ![Front-end application](../assets/img/get-started/front-end-application.png){.cInlineImage-half}
+
+6. Add three new reading items with different statuses.
+
+    For example, the details can be as follows:
+
+     | **Title**                 | **Author**        | **Status** |
+     |---------------------------|-------------------|------------|
+     | `The Museum of Innocence` | `Orhan Pamuk`     | `reading`  |
+     | `The Remains of the Day`  | `Kazuo Ishiguro`  | `to_read`  |
+     | `David Copperfield`       | `Charles Dickens` | `read`     |
+
+    To add each record, follow the steps given below:
+
+    1. Click **+ Add New**.
+
+    2. Enter values for the **Name**, **Author**, and **Status** fields.
+
+    3. Click **Save**.
+
+    Three tabs will appear for each status. To delete a reading list item, you can click **Delete** for it.
+
+8. Log out of the front-end application.
+
+9. Log in to the front-end application again with the credentials of a different user that you created in Asgardeo.
+
+    You will not see the reading list items that you previously entered.
+
+## Step 3: Create and publish a new version of the REST API
+
+Currently, the updates to the readinglist are saved only as long as the deployment status of the `readingList` API remains **Active**. If you undeploy and redeploy the `readingList` API, you cannot retrieve the reading list with the updates you made after the last deployment. To address this, let's connect the REST API to a MySQL database. You can then release the improved REST API as a new version where data persistence is the new feature offered to application users.
```

Comment:
Shall we say why it is not available? because Choreo deploys in a new container?

Reply:
To address this, let's connect the REST API to a MySQL database. You can then release the improved REST API as a new version where data persistence is the new feature offered to application users. -? To persist the data, lets' connect the REST API to a MySQL database. Then let's create and publish a new version of the REST API with the ability to persist the data to the MySQL database.

Reply:
Done